docs/local-development-setup/invite-bot.md

```diff
@@ -0,0 +1,190 @@
+       "DISCORD_PUBLIC_KEY": "***",
+       "ELUVIO_CLIENT_ID": "***",
+       "ELUVIO_CLIENT_SECRET": "***",
+       "ETHERSCAN_API_KEY": "***",
+       "ETHPLORER_API_KEY": "***",
+       "FLOW_NETWORK": "***",
+       "GITHUB_CLIENT_ID": "***",
+       "GITHUB_CLIENT_SECRET": "***",
+       "INFURA_PROJECT_ID": "***",
+       "INFURA_PROJECT_SECRET": "***",
+       "INSTAGRAM_CLIENT_ID": "***",
+       "INSTAGRAM_CLIENT_SECRET": "***",
+       "KCHANNEL_TEST_RECEIVER_MNEMONIC": "***",
+       "KCHANNEL_TEST_SENDER_MNEMONIC": "***",
+       "MAGIC_LINK_SECRET_API_KEY": "***",
+       "MORALIS_API_KEY": "***",
+       "NEAR_MINTER_ACCOUNTS": [
+         {
+           "account_id": "***",
+           "private_key": "***"
+         },
+         {
+           "account_id": "***",
+           "tenant_id": "***",
+           "private_key": "***"
+         },
+         {
+           "account_id": "***",
+           "private_key": "***",
+           "sponsored_amount": 0.1
+         }
+       ],
+       "NEAR_TESTNET_ACCOUNT_ID": "***",
+       "NEAR_TESTNET_ACCOUNT_PRIVATE_KEY": "***",
+       "NEAR_TIPPING_CONFIG": {
+         "contractId": "***",
+         "publicKey": "***",
+         "privateKey": "***",
+         "ethereumPrivateKey": "***"
+       },
+       "NIFTY_CLIENT_ID": "***",
+       "NIFTY_CLIENT_SECRET": "***",
+       "OPENSEA_API_KEY": "***",
+       "OPEN_SEARCH_DOMAIN": "***",
+       "OREID_API_KEY": "***",
+       "OREID_APP_ID": "***",
+       "POCKET_APPLICATION_KEY": "***",
+       "PRIVY_API_KEY": "***",
+       "PRIVY_API_SECRET": "***",
+       "PROMETHEUS_PUSH_GATEWAY": "***",
+       "REDDIT_BOT_APP_ID_COLLABLAND": "***",
+       "REDDIT_BOT_APP_SECRET_COLLABLAND": "***",
+       "REDIS_HOST": "***",
+       "REDIS_PASSWORD": "***",
+       "REDIS_PORT": 6379,
+       "REDIS_USERNAME": "***",
+       "ROLL_REFRESH_TOKEN": "***",
+       "SLACK_CLIENT_ID": "***",
+       "SLACK_CLIENT_SECRET": "***",
+       "SLACK_SIGNING_SECRET": "***",
+       "SOLANA_RPC_URL": "***",
+       "STRIPE_API_KEY": "***",
+       "STRIPE_SIGNING_SECRET": "***",
+       "TELEGRAM_BOT_NAME": "***",
+       "TELEGRAM_BOT_TOKEN": "***",
+       "TEZOS_SECRET_KEY": "***",
+       "TWITTER_CONSUMER_KEY": "***",
+       "TWITTER_CONSUMER_SECRET": "***",
+       "VERAMO_AGENT_URL": "***",
+       "VERAMO_API_KEY": "***"
+     }
+     ```
+
+  4. BUG
+     Describing index keys for [ 'collabland-table-caleb-dev', 'collabland-api-table-caleb-dev' ] +2s
+     Cannot start the application. ResourceNotFoundException: Requested resource not found: Table: collabland-table-caleb-dev not found
+     at deserializeAws_json1_0ResourceNotFoundExceptionResponse (/Users/calebgates/WebstormProjects/CollabLand/dev-x-test-1/collabland-monorepo-dev-x-test-1/node_modules/@aws-sdk/client-dynamodb/dist-cjs/protocols/Aws_json1_0.js:3036:23) 1. Tarmac team needs to rebuild caleb-dev infrastructure because it was created on an old version
```

Comment:
Let's add the solution to a new line. @bhupesh-sf Also, just realized that issues and solutions already existed. Let’s improve these together. The solutions are not helpful currently for all 4 of these.

Reply:
removed these potential errors as they are no more relevant